src/components/organisms/AppRouter/AdminV3Subrouter.tsx

```diff
@@ -30,8 +30,8 @@ export const AdminV3Subrouter: React.FC = () => (
     <Route path={ADMIN_V3_WORLDS_BASE_URL} component={WorldsDashboard} />
     <Route path={ADMIN_V3_VENUE_PARAM_URL} component={AdminVenueView} />
     <Route
-      path={ADMIN_V3_ADVANCED_PARAM_URL}
-      component={AdminAdvancedSettings}
+      path={ADMIN_V3_SPACE_SETTINGS_PARAM_URL}
```

Comment:
Looking top down on the listing here, I can suggest:
-  `AdminSpaceSettings` -> `SpaceEditor` as an analogue to `WorldEditor`, and - `ADMIN_V3_SPACE_SETTINGS_PARAM_URL` -> `ADMIN_V3_SPACE_PARAM_URL` as analogue to `ADMIN_V3_VENUE_PARAM_URL`

But then again, isn't `ADMIN_V3_VENUE_PARAM_URL` the actual `ADMIN_V3_SPACE_PARAM_URL` and `AdminVenueView` the `SpaceEditor`?

Reply:
renamed multiple constants in bf032ac
didn't touch `AdminVenueView` as it's doing different thing than `SpaceEditor`
think we'll get rid of `venue` in naming in some other PR